File: android/BaseAndroidComposeKtor/app/src/main/java/dev/subfly/baseandroidcomposektor/core/domain/model/CoinHomeModel.kt
package dev.subfly.baseandroidcomposektor.core.domain.model

import dev.subfly.baseandroidcomposektor.core.util.CoinType
import dev.subfly.baseandroidcomposektor.core.util.Constants

data class CoinHomeModel(
    val id: String,
    val name: String = Constants.EMPTY_STRING,
    val symbol: String = Constants.EMPTY_STRING,
    val isActive: Boolean = false,
    val isNew: Boolean = false,
    val type: CoinType = CoinType.COIN,
)

File: ios/BaseiOSUIKit/BaseiOSUIKit/Core/Domain/Model/CoinHomeModel.swift
//
//  CoinHomeModel.swift
//  BaseiOSUIKit
//
//  Created by Ali Taha on 5.12.2023.
//

import Foundation

struct CoinHomeModel: Identifiable {
    let id: String
    var name: String = Constants.EMPTY_STRING
    var symbol: String = Constants.EMPTY_STRING
    var isActive: Bool = false
    var isNew: Bool = false
    var type: CoinType = .invalid
}
